src/main/java/org/openelisglobal/dataexchange/fhir/FhirUtil.java

```diff
@@ -43,11 +44,21 @@ public IGenericClient getFhirClient(String fhirStorePath) {
         return fhirClient;
     }
 
-    public IGenericClient getLocalFhirClient() {
-        IGenericClient fhirClient = fhirContext.newRestfulGenericClient(fhirConfig.getLocalFhirStorePath());
+    @Bean(name = "clientRegistryFhirClient")
```

Comment:
@mherman22 we may want to improve how this client is created to deal with instances when the system might be booted up and the client fails to be created because the server is unreacheable. Create a mechanism to reattempt to create the instance if it is null

Reply:
alright thanks! lemi do that